content/en/docs/reference/kubectl/docker-cli-to-kubectl.md

```diff
@@ -1,16 +1,20 @@
 ---
 title: kubectl for Docker Users
+content_template: templates/concept
 reviewers:
 - bgrant0607
 - brendandburns
 - thockin
 ---
 
-You can use the Kubernetes command line tool kubectl to interact with the api. You can use kubectl if you are familiar with docker-cli. However, there are a few differences in the docker-cli commands and the kubectl commands. Each of the following section details a docker subcommand and explains the kubectl equivalent.
-
 {{< toc >}}
 
```

Comment:
Is the toc shortcode needed?

Reply:
+1 to removing toc shortcode; should not be needed.

Reply:
I removed it, thanks.